doc/help/install-config-file.md

```diff
@@ -0,0 +1,26 @@
+install-config-file(1) -- Install configuration files in Elektra
+
+## SYNOPSIS
+
+`install-config-file <elektra path> <config file> [<format>]`
+
+`<elektra path>` is a path in Elektra
+`<config file>` is some file on the filesystem
+
+`line` will be used if `<format>` is unspecified
+
+The script has to be called as administrator
```

Comment:
[Here](https://github.com/ElektraInitiative/libelektra/pull/3106#discussion_r339964670) you mentioned documentation for `install-config-file` in the tutorial. For me it would make sense to include all examples that concern only the script to the man page and mention in the tutorial only very shortly that the script exists while referring to its man page. What do you think about this?

Reply:
Tutorials and man pages have different purposes but it all depends on how it is written. It would be good to have a guide how to use `install-config-file` for your application. And for this a tutorial seems to be the better style.

Reply:
I've now added both a tutorial (with the intent "This takes you step by step through a use case for the script) and an example for the man page (only one line with the intent "This is what a valid call looks like).